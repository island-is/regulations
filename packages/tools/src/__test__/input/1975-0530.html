<div class="Section1">
  <p align="center">1. gr.</p>

  <p align="justify">Innheimtumenn ríkissjóðs skulu gera ríkissjóði reikningsskil mánaðarlega fyrir þeim innheimtum sem þeim er trúað fyrir.</p>

  <p align="justify">Með innheimtumönnum ríkissjóðs er í reglugerð þessari átt við sýslumenn, bæjarfógeta, lögreglustjóra, tollstjóra, borgarfógeta, borgardómara og sakadómara, og eru þeir í reglugerðinni einu orði nefndir innheimtumenn ríkissjóðs.</p>

  <p align="justify">Aðrir innheimtumenn ríkissjóðs skulu hlíta ákvæðum reglugerðar þessarar, eftir því sem við á.</p>

  <p align="center">2. gr.</p>

  <p align="justify">Eftirtaldar bækur skulu færðar hjá þeim innheimtumönnum ríkissjóðs sem hafa handfært bókhald:</p>

  <ol>
  <li>
  Aðalsjóðbók með þessum undirsjóðbókum:

  <ol type="a">
  <li>Manntalsgjaldasjóðbók.</li>

  <li>Tollabók.</li>

  <li>Aukatekjubók.</li>

  <li>Útflutningsbók.</li>

  <li>Skemmtanaskattsbók.</li>

  <li>Vörugjaldabók.</li>
  </ol>
  </li>

  <li>Tryggingasjóðbók.</li>

  <li>Bifreiðaskattsbók.</li>

  <li>Viðskiptamannabók.</li>

  <li>Birgðabækur.</li>
  </ol>

  <p align="justify">Bækur þessar skulu færðar í samræmi við fyrirmæli frá ríkisbókhaldinu. Skulu þær löggiltar af fjármálaráðuneytinu.</p>

  <p align="justify">Þeir innheimtumenn ríkissjóðs sem hafa vélfært bókhald skulu færa það að fyrirmælum ríki~bókhaldsins enda komi þar fram sömu upplýsingar og krafist er við handfært bókhald.</p>

  <p align="center">3. gr.</p>

  <p align="justify">Í aðalsjóðbók skal færa þessa reikninga:</p>

  <ol>
  <li>Sjóðreikningur.</li>

  <li>Bankareikningur.</li>

  <li>Ríkissjóður.</li>

  <li>Tryggingastofnun og tryggingaumboð.</li>

  <li>Greiðslur fyrir ráðuneyti.</li>

  <li>Viðskiptamenn.</li>

  <li>Kostnaður við embættið.</li>

  <li>Geymslufjárreikningur.</li>

  <li>Sýslu- og sýsluvegasjóður.</li>
  </ol>

  <p align="justify">Færa skal í lok hvers mánaðar úr undirsjóðbókum í aðalsjóðbók allt það, sem samkvæmt þeim hefur verið innheimt.</p>

  <p align="center">4. gr.</p>

  <p align="justify">Í manntalsgjaldasjóðbók skal færa innheimtu gjalda þannig sundurliðaða:</p>

  <ol>
  <li>Þinggjöld yfirstandandi árs.</li>

  <li>Eftirstöðvar þinggjalda.</li>

  <li>Sölugjald af vörusölu og þjónustu.</li>

  <li>Bifreiðaskattur.</li>

  <li>Iðgjöld bifreiðastjóra.</li>

  <li>Skoðunargjald bifreiða.</li>

  <li>Öryggiseftirlitsgjald.</li>

  <li>Afgjöld ríkisjarða og lóðaleigur.</li>

  <li>Iðnlánasjóðsgjald.</li>

  <li>10. Launaskattur.</li>

  <li>11. Þungaskattur af bifreiðum með ökumæla.</li>

  <li>12. Skipulagsgjald.</li>

  <li>13. Skoðunargjald skipa.</li>
  </ol>

  <p align="justify">Innheimt þinggjöld skal daglega færa ósundurliðuð í manntalsgjaldasjóðbók.</p>

  <p align="justify">Í lok hvers mánaðar skal sundurliða heildarfjárhæð innheimtra þinggjalda samkvæmt hlutfallstölum sem ákvarðaðar séu i samráði við ríkisbókhaldið. Sundurliðunin skal sýnd í manntalsgjaldasjóðbók.</p>

  <p align="center">5. gr.</p>

  <p align="justify">Í tollabók skal færa sundurliðaða innheimtu aðflutningsgjalda.</p>

  <p align="center">6. gr.</p>

  <p align="justify">Í aukatekjubók skal færa sundurliðaða innheimtu þannig:</p>

  <ol>
  <li>Þinglýsingargjald.</li>

  <li>Stimpilgjald.</li>

  <li>Skipagjöld.</li>

  <li>Almennar aukatekjur.</li>

  <li>Skrásetningargjald bifreiða.</li>

  <li>Erfðafjárskattur.</li>

  <li>Vörugjald af innlendum framleiðsluvörum.</li>

  <li>Skemmtanaskattur.</li>

  <li>Prófgjöld.</li>

  <li>Sektir.</li>
  </ol>

  <p align="justify">Skipagjöld, prófgjöld og sektir skal sundurliða í aukatekjubók í lok hvers mánaðar.</p>

  <p align="justify">Skipagjöld sundurliðist þannig:</p>

  <ol>
  <li>Afgreiðslugjald skipa.</li>

  <li>Vitagjald.</li>

  <li>Sóttvarnarsjóðsgjald.</li>

  <li>Áritunargjald.</li>
  </ol>

  <p align="justify">Prófgjöld sundurliðist þannig:</p>

  <ol>
  <li>Prófgjöld ökumanna.</li>

  <li>Prófgjöld iðnnema.</li>
  </ol>

  <p align="justify">Sektir sundurliðist þannig:</p>

  <ol>
  <li>Ríkissjóðssektir.</li>

  <li>Landhelgissjóðssektir.</li>
  </ol>

  <p align="center">7. gr.</p>

  <p align="justify">Í útflutningsgjaldabók skal færa öll gjöld af útflutningi.</p>

  <p align="center">8. gr.</p>

  <p align="justify">Í skemmtanaskattsbók skal færa öll gjöld sem innheimta ber af skemmtanahaldi.</p>

  <p align="center">9. gr.</p>

  <p align="justify">Í vörugjaldabók skal færa öll vörugjöld af innlendri framleiðslu.</p>

  <p align="center">10. gr.</p>

  <p align="justify">Í tryggingasjóðbók skal færa eftirtalda reikninga:</p>

  <ol>
  <li>Sjóður.</li>

  <li>Bankareikningur.</li>

  <li>Embættisreikningur.</li>

  <li>Framlag sveitarfélaga.</li>

  <li>Barnalífeyrir.</li>

  <li>Ellilífeyrir</li>

  <li>Örorkulífeyrir.</li>

  <li>Fæðingarstyrkur.</li>

  <li>Ekkjubætur.</li>

  <li>10. Ekkjulífeyrir.</li>

  <li>11. Mæðralaun.</li>

  <li>12. Makabætur.</li>
  </ol>

  <p align="justify">Ellilífeyri og örorkulífeyri skal sundurliða mánaðarlega í tryggingasjóðbók í lok hvers mánaðar.</p>

  <p align="justify">Ellilífeyrir sundurliðist þannig:</p>

  <ol>
  <li>Ellilífeyrir.</li>

  <li>Tekjutryggingauppbót.</li>

  <li>Heimildaruppbót sbr. 2. mgr. 19. gr.</li>
  </ol>

  <p align="justify">Örorkulífeyrir sundurliðist þannig:</p>

  <ol>
  <li>Örorkulífeyrir.</li>

  <li>Tekjutryggingauppbót.</li>

  <li>Heimildaruppbót sbr. 2. mgr. 19. gr.</li>
  </ol>

  <p align="center">11. gr.</p>

  <p align="justify">Í bifreiðaskattsbók skal færa gjöld þau, sem greiða ber við bifreiðaskoðun. Greiðslur á bifreiðaskatti skal ,jafnframt færa í manntalsgjaldasjóðbók.</p>

  <p align="center">12. gr.</p>

  <p align="justify">Færa skal viðskiptamannabók er svarar til dálksins "viðskiptamenn", í aðalsjóðbók.</p>

  <p align="center">13. gr.</p>

  <p align="justify">Innheimtumenn ríkissjóðs skulu halda birgðabækur yfir stimpilmerki og önnur verðmæti, sem í vörslum þeirra eru, svo sem bifreiðamerki, ökuskírteini, viðskiptabækur sjómanna o. s. frv. Skilagrein fyrir verðmætum þessum skal árlega senda ríkisendurskoðuninni og andvirði þeirra skal skilað eftir því sem sérstaklega er fyrirmælt.</p>

  <p align="center">14. gr.</p>

  <p align="justify">Um hver mánaðamót og í árslok ber innheimtumönnum ríkissjóðs að senda ríkisbókhaldi og ríkisendurskoðun yfirlit um innheimtu í því formi sem ríkisbókhald og ríkisendurskoðun ákveða.</p>

  <p align="justify">1. Mánaðarinnheimtuskilagrein skal senda ríkisbókhaldi i tveimur eintökum, eigi síðar en á þriðja virkum degi næsta mánaðar. Ríkisbókhaldið framsendir annað eintakið til ríkisendurskoðunar.</p>

  <p align="justify">2. Ráðstöfunaryfirlit mánaðarins skal senda ríkisbókhaldi í tveimur eintökum eigi siðar en 7. dag næsta mánaðar. Ríkisbókhald framsendir annað eintakið til ríkisendurskoðunar.</p>

  <p align="justify">Í árslok ber innheimtumönnum að semja heildarskilagrein og senda ríkisbókhaldi og ríkisendurskoðun sitt hvort eintakið. Enn fremur skal senda ríkisendurskoðun með heildarskilagrein embættisbækur þær og skilagreinar, sem hún tiltekur.</p>

  <p align="justify">Reikningum embættisins ber að loka 31. desember ár hvert. Heildarskilagrein ber að skila eigi síðar en 20. febrúar.</p>

  <p align="center">15. gr.</p>

  <p align="justify">Innheimtumenn ríkissjóðs skulu gera skil á innheimtufé ríkissjóðs, samkvæmt nánari fyrirmælum fjármálaráðuneytisins.</p>

  <p align="center">16. gr.</p>

  <p align="justify">Innheimtumönnum ríkissjóðs ber að gera sjóðsuppgjör daglega. Þeir skulu geyma og ávaxta fé embættisins í ríkisbanka, þar til því er skilað. Ef ekki er ríkisbanki á staðnum, en hins vegar annar banki eða sparisjóður, skulu innheimtumenn ríkissjóðs, að fengnu samþykki fjármálaráðuneytisins, geyma þar og ávaxta fé embættisins.</p>

  <p align="justify">Fé embættisins er allt það- fé, sem innheimtumenn ríkissjóðs taka við samkvæmt embættisskyldu, hvort sem það á að renna í ríkissjóð eða til annarra. Fé búa og sjóða skal vera á sérreikningi fyrir hvert bú og hvern sjóð.</p>

  <p align="justify">Vexti af fé búa og sjóða ber að færa hverju þeirra til tekna þegar er þeir til falla. Vexti af öðru embættisfé skal færa ríkissjóði til tekna og skila þeim með öðrum tekjum.</p>

  <p align="center">17. gr.</p>

  <p align="justify">Innheimtumenn ríkissjóðs skulu heimta inn tekjur þær, sem þeim er falin innheimta á, svo fljótt sem auðið er eftir að þær falla í gjalddaga.</p>

  <p align="justify">Þeim ber að neyta lögtaks eða annarra lögheimilaðra aðfarargerða áður en frestur til þeirra er liðinn.</p>

  <p align="center">18. gr.</p>

  <p align="justify">Innheimtumönnum ríkissjóðs ber að veita starfsmönnum ríkisendurskoðunarinnar frjálsan og óhindraðan aðgang að öllum embættisbókum og skjölum og gefa þeim allar þær upplýsingar, sem þeir æskja um fjárreiður embættisins.</p>

  <p align="justify">Innheimtumönnum ríkissjóðs ber að framvísa sjóði embættisins hvenær sem endurskoðunin krefst þess, og skal kann á hverjum tíma vera sá, sem sjóðbækur sýna. Þegar sjóðtalning fer fram á vegum ríkisendurskoðunarinnar ber innheimtumanni ríkissjóðs að sýna endurskoðendum allar fjárreiður, sem harm hefur undir höndum, einnig þær, sem ekki koma ríkissjóði við, til þess að gengið verði úr skugga um það, að ekkert fé fari á milli sjóða.</p>

  <p align="center">19. gr.</p>

  <p align="justify">Ef innheimtumenn ríkissjóðs greiða ekki tekjur þær, er þeir eiga að standa skil á til ríkissjóðs, samkvæmt fyrirmælum fjármálaráðuneytisins sbr. 15. gr., eða gera ekki reikning í tæka tíð samkvæmt 14. gr., eða standa í skuld samkvæmt skilagrein þeirri, sem fram er komin, eða ef sjóðþurrð reynist hjá einhverjum embættismanni ríkissjóðs, verður farið eftir ákvæðum III. kafla laga nr. 38 14. apríl 1954, um réttindi og skyldur starfsmanna ríkisins.</p>

  <p align="justify">Reglugerð þessi er sett samkvæmt lögum um ríkisbókhald, gerð ríkisreiknings og fjárlaga nr. 52 13. maí 1966 og öðlast gildi 1. janúar 1976. Með henni er úr gildi numin reglugerð um opinber reikningsskil dags. 18. nóvember 1955, sem auglýst er í B-deild Stjórnartíðinda nr. 134 á því ári.</p>

  <p align="center"><em>Fjármálaráðuneytið, 17. desember 1975.</em></p>

  <p align="center"><strong>Matthías Á. Mathiesen.</strong></p>

  <p align="right"><em>Höskuldur Jónsson.</em></p>
  </div>
