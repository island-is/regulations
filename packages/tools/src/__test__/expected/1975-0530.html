<h3 class="article__title">
1.
gr.
</h3>

<p>
Innheimtumenn
ríkissjóðs
skulu
gera
ríkissjóði
reikningsskil
mánaðarlega
fyrir
þeim
innheimtum
sem
þeim
er
trúað
fyrir.
</p>

<p>
Með
innheimtumönnum
ríkissjóðs
er
í
reglugerð
þessari
átt
við
sýslumenn,
bæjarfógeta,
lögreglustjóra,
tollstjóra,
borgarfógeta,
borgardómara
og
sakadómara,
og
eru
þeir
í
reglugerðinni
einu
orði
nefndir
innheimtumenn
ríkissjóðs.
</p>

<p>
Aðrir
innheimtumenn
ríkissjóðs
skulu
hlíta
ákvæðum
reglugerðar
þessarar,
eftir
því
sem
við
á.
</p>

<h3 class="article__title">
2.
gr.
</h3>

<p>
Eftirtaldar
bækur
skulu
færðar
hjá
þeim
innheimtumönnum
ríkissjóðs
sem
hafa
handfært
bókhald:
</p>

<ol>

<li>

<p>
Aðalsjóðbók
með
þessum
undirsjóðbókum:
</p>

<ol type="a">

<li>
Manntalsgjaldasjóðbók.
</li>

<li>
Tollabók.
</li>

<li>
Aukatekjubók.
</li>

<li>
Útflutningsbók.
</li>

<li>
Skemmtanaskattsbók.
</li>

<li>
Vörugjaldabók.
</li>

</ol>

</li>

<li>
Tryggingasjóðbók.
</li>

<li>
Bifreiðaskattsbók.
</li>

<li>
Viðskiptamannabók.
</li>

<li>
Birgðabækur.
</li>

</ol>

<p>
Bækur
þessar
skulu
færðar
í
samræmi
við
fyrirmæli
frá
ríkisbókhaldinu.
Skulu
þær
löggiltar
af
fjármálaráðuneytinu.
</p>

<p>
Þeir
innheimtumenn
ríkissjóðs
sem
hafa
vélfært
bókhald
skulu
færa
það
að
fyrirmælum
ríki~bókhaldsins
enda
komi
þar
fram
sömu
upplýsingar
og
krafist
er
við
handfært
bókhald.
</p>

<h3 class="article__title">
3.
gr.
</h3>

<p>
Í
aðalsjóðbók
skal
færa
þessa
reikninga:
</p>

<ol>

<li>
Sjóðreikningur.
</li>

<li>
Bankareikningur.
</li>

<li>
Ríkissjóður.
</li>

<li>
Tryggingastofnun
og
tryggingaumboð.
</li>

<li>
Greiðslur
fyrir
ráðuneyti.
</li>

<li>
Viðskiptamenn.
</li>

<li>
Kostnaður
við
embættið.
</li>

<li>
Geymslufjárreikningur.
</li>

<li>
Sýslu-
og
sýsluvegasjóður.
</li>

</ol>

<p>
Færa
skal
í
lok
hvers
mánaðar
úr
undirsjóðbókum
í
aðalsjóðbók
allt
það,
sem
samkvæmt
þeim
hefur
verið
innheimt.
</p>

<h3 class="article__title">
4.
gr.
</h3>

<p>
Í
manntalsgjaldasjóðbók
skal
færa
innheimtu
gjalda
þannig
sundurliðaða:
</p>

<ol>

<li>
Þinggjöld
yfirstandandi
árs.
</li>

<li>
Eftirstöðvar
þinggjalda.
</li>

<li>
Sölugjald
af
vörusölu
og
þjónustu.
</li>

<li>
Bifreiðaskattur.
</li>

<li>
Iðgjöld
bifreiðastjóra.
</li>

<li>
Skoðunargjald
bifreiða.
</li>

<li>
Öryggiseftirlitsgjald.
</li>

<li>
Afgjöld
ríkisjarða
og
lóðaleigur.
</li>

<li>
Iðnlánasjóðsgjald.
</li>

<li>
10.
Launaskattur.
</li>

<li>
11.
Þungaskattur
af
bifreiðum
með
ökumæla.
</li>

<li>
12.
Skipulagsgjald.
</li>

<li>
13.
Skoðunargjald
skipa.
</li>

</ol>

<p>
Innheimt
þinggjöld
skal
daglega
færa
ósundurliðuð
í
manntalsgjaldasjóðbók.
</p>

<p>
Í
lok
hvers
mánaðar
skal
sundurliða
heildarfjárhæð
innheimtra
þinggjalda
samkvæmt
hlutfallstölum
sem
ákvarðaðar
séu
i
samráði
við
ríkisbókhaldið.
Sundurliðunin
skal
sýnd
í
manntalsgjaldasjóðbók.
</p>

<h3 class="article__title">
5.
gr.
</h3>

<p>
Í
tollabók
skal
færa
sundurliðaða
innheimtu
aðflutningsgjalda.
</p>

<h3 class="article__title">
6.
gr.
</h3>

<p>
Í
aukatekjubók
skal
færa
sundurliðaða
innheimtu
þannig:
</p>

<ol>

<li>
Þinglýsingargjald.
</li>

<li>
Stimpilgjald.
</li>

<li>
Skipagjöld.
</li>

<li>
Almennar
aukatekjur.
</li>

<li>
Skrásetningargjald
bifreiða.
</li>

<li>
Erfðafjárskattur.
</li>

<li>
Vörugjald
af
innlendum
framleiðsluvörum.
</li>

<li>
Skemmtanaskattur.
</li>

<li>
Prófgjöld.
</li>

<li>
Sektir.
</li>

</ol>

<p>
Skipagjöld,
prófgjöld
og
sektir
skal
sundurliða
í
aukatekjubók
í
lok
hvers
mánaðar.
</p>

<p>
Skipagjöld
sundurliðist
þannig:
</p>

<ol>

<li>
Afgreiðslugjald
skipa.
</li>

<li>
Vitagjald.
</li>

<li>
Sóttvarnarsjóðsgjald.
</li>

<li>
Áritunargjald.
</li>

</ol>

<p>
Prófgjöld
sundurliðist
þannig:
</p>

<ol>

<li>
Prófgjöld
ökumanna.
</li>

<li>
Prófgjöld
iðnnema.
</li>

</ol>

<p>
Sektir
sundurliðist
þannig:
</p>

<ol>

<li>
Ríkissjóðssektir.
</li>

<li>
Landhelgissjóðssektir.
</li>

</ol>

<h3 class="article__title">
7.
gr.
</h3>

<p>
Í
útflutningsgjaldabók
skal
færa
öll
gjöld
af
útflutningi.
</p>

<h3 class="article__title">
8.
gr.
</h3>

<p>
Í
skemmtanaskattsbók
skal
færa
öll
gjöld
sem
innheimta
ber
af
skemmtanahaldi.
</p>

<h3 class="article__title">
9.
gr.
</h3>

<p>
Í
vörugjaldabók
skal
færa
öll
vörugjöld
af
innlendri
framleiðslu.
</p>

<h3 class="article__title">
10.
gr.
</h3>

<p>
Í
tryggingasjóðbók
skal
færa
eftirtalda
reikninga:
</p>

<ol>

<li>
Sjóður.
</li>

<li>
Bankareikningur.
</li>

<li>
Embættisreikningur.
</li>

<li>
Framlag
sveitarfélaga.
</li>

<li>
Barnalífeyrir.
</li>

<li>
Ellilífeyrir
</li>

<li>
Örorkulífeyrir.
</li>

<li>
Fæðingarstyrkur.
</li>

<li>
Ekkjubætur.
</li>

<li>
10.
Ekkjulífeyrir.
</li>

<li>
11.
Mæðralaun.
</li>

<li>
12.
Makabætur.
</li>

</ol>

<p>
Ellilífeyri
og
örorkulífeyri
skal
sundurliða
mánaðarlega
í
tryggingasjóðbók
í
lok
hvers
mánaðar.
</p>

<p>
Ellilífeyrir
sundurliðist
þannig:
</p>

<ol>

<li>
Ellilífeyrir.
</li>

<li>
Tekjutryggingauppbót.
</li>

<li>
Heimildaruppbót
sbr.
2.
mgr.
19.
gr.
</li>

</ol>

<p>
Örorkulífeyrir
sundurliðist
þannig:
</p>

<ol>

<li>
Örorkulífeyrir.
</li>

<li>
Tekjutryggingauppbót.
</li>

<li>
Heimildaruppbót
sbr.
2.
mgr.
19.
gr.
</li>

</ol>

<h3 class="article__title">
11.
gr.
</h3>

<p>
Í
bifreiðaskattsbók
skal
færa
gjöld
þau,
sem
greiða
ber
við
bifreiðaskoðun.
Greiðslur
á
bifreiðaskatti
skal
,jafnframt
færa
í
manntalsgjaldasjóðbók.
</p>

<h3 class="article__title">
12.
gr.
</h3>

<p>
Færa
skal
viðskiptamannabók
er
svarar
til
dálksins
"viðskiptamenn",
í
aðalsjóðbók.
</p>

<h3 class="article__title">
13.
gr.
</h3>

<p>
Innheimtumenn
ríkissjóðs
skulu
halda
birgðabækur
yfir
stimpilmerki
og
önnur
verðmæti,
sem
í
vörslum
þeirra
eru,
svo
sem
bifreiðamerki,
ökuskírteini,
viðskiptabækur
sjómanna
o.
s.
frv.
Skilagrein
fyrir
verðmætum
þessum
skal
árlega
senda
ríkisendurskoðuninni
og
andvirði
þeirra
skal
skilað
eftir
því
sem
sérstaklega
er
fyrirmælt.
</p>

<h3 class="article__title">
14.
gr.
</h3>

<p>
Um
hver
mánaðamót
og
í
árslok
ber
innheimtumönnum
ríkissjóðs
að
senda
ríkisbókhaldi
og
ríkisendurskoðun
yfirlit
um
innheimtu
í
því
formi
sem
ríkisbókhald
og
ríkisendurskoðun
ákveða.
</p>

<p>
1.
Mánaðarinnheimtuskilagrein
skal
senda
ríkisbókhaldi
i
tveimur
eintökum,
eigi
síðar
en
á
þriðja
virkum
degi
næsta
mánaðar.
Ríkisbókhaldið
framsendir
annað
eintakið
til
ríkisendurskoðunar.
</p>

<p>
2.
Ráðstöfunaryfirlit
mánaðarins
skal
senda
ríkisbókhaldi
í
tveimur
eintökum
eigi
siðar
en
7.
dag
næsta
mánaðar.
Ríkisbókhald
framsendir
annað
eintakið
til
ríkisendurskoðunar.
</p>

<p>
Í
árslok
ber
innheimtumönnum
að
semja
heildarskilagrein
og
senda
ríkisbókhaldi
og
ríkisendurskoðun
sitt
hvort
eintakið.
Enn
fremur
skal
senda
ríkisendurskoðun
með
heildarskilagrein
embættisbækur
þær
og
skilagreinar,
sem
hún
tiltekur.
</p>

<p>
Reikningum
embættisins
ber
að
loka
31.
desember
ár
hvert.
Heildarskilagrein
ber
að
skila
eigi
síðar
en
20.
febrúar.
</p>

<h3 class="article__title">
15.
gr.
</h3>

<p>
Innheimtumenn
ríkissjóðs
skulu
gera
skil
á
innheimtufé
ríkissjóðs,
samkvæmt
nánari
fyrirmælum
fjármálaráðuneytisins.
</p>

<h3 class="article__title">
16.
gr.
</h3>

<p>
Innheimtumönnum
ríkissjóðs
ber
að
gera
sjóðsuppgjör
daglega.
Þeir
skulu
geyma
og
ávaxta
fé
embættisins
í
ríkisbanka,
þar
til
því
er
skilað.
Ef
ekki
er
ríkisbanki
á
staðnum,
en
hins
vegar
annar
banki
eða
sparisjóður,
skulu
innheimtumenn
ríkissjóðs,
að
fengnu
samþykki
fjármálaráðuneytisins,
geyma
þar
og
ávaxta
fé
embættisins.
</p>

<p>
Fé
embættisins
er
allt
það-
fé,
sem
innheimtumenn
ríkissjóðs
taka
við
samkvæmt
embættisskyldu,
hvort
sem
það
á
að
renna
í
ríkissjóð
eða
til
annarra.
Fé
búa
og
sjóða
skal
vera
á
sérreikningi
fyrir
hvert
bú
og
hvern
sjóð.
</p>

<p>
Vexti
af
fé
búa
og
sjóða
ber
að
færa
hverju
þeirra
til
tekna
þegar
er
þeir
til
falla.
Vexti
af
öðru
embættisfé
skal
færa
ríkissjóði
til
tekna
og
skila
þeim
með
öðrum
tekjum.
</p>

<h3 class="article__title">
17.
gr.
</h3>

<p>
Innheimtumenn
ríkissjóðs
skulu
heimta
inn
tekjur
þær,
sem
þeim
er
falin
innheimta
á,
svo
fljótt
sem
auðið
er
eftir
að
þær
falla
í
gjalddaga.
</p>

<p>
Þeim
ber
að
neyta
lögtaks
eða
annarra
lögheimilaðra
aðfarargerða
áður
en
frestur
til
þeirra
er
liðinn.
</p>

<h3 class="article__title">
18.
gr.
</h3>

<p>
Innheimtumönnum
ríkissjóðs
ber
að
veita
starfsmönnum
ríkisendurskoðunarinnar
frjálsan
og
óhindraðan
aðgang
að
öllum
embættisbókum
og
skjölum
og
gefa
þeim
allar
þær
upplýsingar,
sem
þeir
æskja
um
fjárreiður
embættisins.
</p>

<p>
Innheimtumönnum
ríkissjóðs
ber
að
framvísa
sjóði
embættisins
hvenær
sem
endurskoðunin
krefst
þess,
og
skal
kann
á
hverjum
tíma
vera
sá,
sem
sjóðbækur
sýna.
Þegar
sjóðtalning
fer
fram
á
vegum
ríkisendurskoðunarinnar
ber
innheimtumanni
ríkissjóðs
að
sýna
endurskoðendum
allar
fjárreiður,
sem
harm
hefur
undir
höndum,
einnig
þær,
sem
ekki
koma
ríkissjóði
við,
til
þess
að
gengið
verði
úr
skugga
um
það,
að
ekkert
fé
fari
á
milli
sjóða.
</p>

<h3 class="article__title">
19.
gr.
</h3>

<p>
Ef
innheimtumenn
ríkissjóðs
greiða
ekki
tekjur
þær,
er
þeir
eiga
að
standa
skil
á
til
ríkissjóðs,
samkvæmt
fyrirmælum
fjármálaráðuneytisins
sbr.
15.
gr.,
eða
gera
ekki
reikning
í
tæka
tíð
samkvæmt
14.
gr.,
eða
standa
í
skuld
samkvæmt
skilagrein
þeirri,
sem
fram
er
komin,
eða
ef
sjóðþurrð
reynist
hjá
einhverjum
embættismanni
ríkissjóðs,
verður
farið
eftir
ákvæðum
III.
kafla
laga
nr.
38
14.
apríl
1954,
um
réttindi
og
skyldur
starfsmanna
ríkisins.
</p>

<p>
Reglugerð
þessi
er
sett
samkvæmt
lögum
um
ríkisbókhald,
gerð
ríkisreiknings
og
fjárlaga
nr.
52
13.
maí
1966
og
öðlast
gildi
1.
janúar
1976.
Með
henni
er
úr
gildi
numin
reglugerð
um
opinber
reikningsskil
dags.
18.
nóvember
1955,
sem
auglýst
er
í
B-deild
Stjórnartíðinda
nr.
134
á
því
ári.
</p>

<p class="Dags" align="center">
<em>Fjármálaráðuneytið,
17.
desember
1975.</em>
</p>

<p align="center">
<strong>Matthías
Á.
Mathiesen.</strong>
</p>

<p align="right">
<em>Höskuldur
Jónsson.</em>
</p>
